### ISSUE TITLE
Remodify the loss

### ISSUE THREAD

**Issue by LittleFish-Coder** [LittleFish-Coder/GemGNN#22]
We actually found that we only use cross entropy to get a good performance so no longer use the combined loss, just use the crossentropy and with val_loss < 0.3 to stop training.

You should change the introduction, abstract, results part, and any related parts in our thesis
